silverorange/Cme
# add cme quiz analytics

To Test:

1. Check out this PR
2. Start emrap with `yarn start --symlinks=cme`
3. Visit `account/cme`
4. Click on Print Certificate on one of the quizzes you've completed
5. View source and search for `CME_Printed` and check that the id is set
6. For the other javascript you'll need to add console.log statements where the amplitude gets called to see it log the id to the console when the button is clicked
